src/GithubFilesBundle/Service/GithubImport.php

```diff
@@ -144,10 +137,11 @@ public function parseGithubStandard(LsDoc $lsDoc, $lsItemKeys, $data){
         $lsAssociation->setLsDoc($lsDoc);
         $lsAssociation->setOrigin($lsItem);
         $lsAssociation->setType(LsAssociation::EXACT_MATCH_OF);
-        $lsAssociation->setDestinationNodeIdentifier($lsItemIdentifier);
+        $lsAssociation->setDestinationNodeIdentifier($lsItem->getIdentifier());
```

Comment:
I'm not sure creating this association makes sense.

Basically it is saying that the lsItem is a match to itself and does not add any other information.